MekHQ/src/mekhq/campaign/mission/AtBDynamicScenarioFactory.java

```diff
@@ -492,15 +659,13 @@ public static int generateForce(AtBDynamicScenario scenario, AtBContract contrac
                     // Configure *all* generated units with appropriate munitions (for BV calcs)
                     Game cGame = campaign.getGame();
                     TeamLoadoutGenerator tlg = new TeamLoadoutGenerator(cGame);
-                    ArrayList<Entity> arrayGeneratedLance = new ArrayList<Entity>(generatedLance);
+                    ArrayList<Entity> arrayGeneratedLance = new ArrayList<>(generatedLance);
                     // bin fill ratio will be adjusted by the loadout generator based on piracy and quality
                     ReconfigurationParameters rp = TeamLoadoutGenerator.generateParameters(
                             cGame,
                             cGame.getOptions(),
                             arrayGeneratedLance,
-                            factionCode,
-                            new ArrayList<Entity>(),
-                            new ArrayList<String>(),
+                            factionCode, new ArrayList<>(), new ArrayList<>(),
```

Comment:
Minor stylistic nit: if arguments are expanded into a vertical list, each should be on a separate line so that the eye does not skip over them (as I did here, briefly thinking the `new ArrayList<>()` args had gone missing).

Reply:
Ah, that's a 'My Bad'. I'll fix it in a sec :)